proto/testing/BUILD.bazel

```diff
@@ -14,13 +14,33 @@ go_proto_library(
     name = "ethereum_testing_go_proto",
     compiler = "//:proto_compiler",
     importpath = "github.com/prysmaticlabs/prysm/proto/testing",
-    proto = ":ethereum_testing_proto",
+    proto = ":testing_proto",
     visibility = ["//visibility:public"],
 )
 
 go_library(
     name = "go_default_library",
+    srcs = ["ssz_minimal.yaml.go"],
     embed = [":ethereum_testing_go_proto"],
     importpath = "github.com/prysmaticlabs/prysm/proto/testing",
     visibility = ["//visibility:public"],
 )
+
+go_test(
+    name = "go_default_test",
+    srcs = ["ssz_compatibility_test.go"],
+    embed = [":go_default_library"],
+    deps = [
+        "//proto/beacon/p2p/v1:go_default_library",
+        "//shared/testutil:go_default_library",
+        "@com_github_ghodss_yaml//:go_default_library",
+        "@com_github_prysmaticlabs_go_ssz//:go_default_library",
+    ],
+)
+
+proto_library(
+    name = "testing_proto",
```

Comment:
Why did you add this target? It's identical to ethereum_testing_proto

Reply:
The lint was complaining for `ethereal_testing_proto` (not suppose to have _), we can use `ethereumtesting` if you want

```
- package ethereum.testing;
+ package testing;
```

Reply:
OK, please remove the other target in that case.